instance.tf

```diff
@@ -1,8 +1,8 @@
 resource "google_sql_database_instance" "instance" {
   database_version     = var.database_version
-  name                 = var.instance_name
+  name                 = "${var.project_prefix}-${var.instance_name}"
   deletion_protection  = var.deletion_protection
-  master_instance_name = var.primary_instance_name
+  master_instance_name = "${var.primary_instance_name}" != null ? "${var.project_prefix}-${var.primary_instance_name}" : null
```

Comment:
I kinda doubt whether we should add the prefix here as well. Only other use case would be "cross project" replication 🤔

Reply:
Is it not good to have it if we use replication in future or it is the case we will never you cross project replication. Then i can remove prefix thing for primary instance.

Reply:
I was mostly just "thinking out loud".

Guess we should leave it like this, for now. Could always introduce something like `external_primary_instance_name` with more flexibility down the road, if we'd ever need that.